backend/fms_core/services/study.py

```diff
@@ -0,0 +1,39 @@
+from django.core.exceptions import ValidationError
+from fms_core.models import Study
+
+def create_study(letter, project, worfklow, start, end, reference_genome=None):
```

Comment:
Along with my comment about moving letter generation i would remove the parameter altogether. As long as we cannot delete studies this will be ok. Once we can delete we'll have to change to an algorithm that pick the highest letter then add one (maybe we should do that right away). In any case the choice of letter should never be given to the caller i think.